src/components/HardwareListing/HardwareListing.tsx

```diff
@@ -56,7 +56,7 @@ const OnItemDragStart = (
   };
 };
 
-const ModuleGroup = (props: { level: number; group: ListingGroup; hideGroup?: boolean }) => {
+const ModuleGroup = (props: { level: number; group: ListingGroup; hideGroup?: number }) => {
```

Comment:
Does the name `hideGroup` still make sense? `hideGroupDepth`? I dunno .. I hate picking names :)